src/appsco-app.html

```diff
@@ -2680,15 +2734,40 @@
                 this._notify('An error occurred. Selected accounts were not removed from company. Please try again.');
```

Comment:
Although parameters in JS functions are optional, we could add it for the sake of code readability. I'll add second parameter where missing.